### ISSUE TITLE
Consider using infinite parking for some proposals

### ISSUE THREAD

**Issue by dabreegster** [a-b-street/abstreet#688]
The Broadmoor access proposal winds up with this overall trip time distribution:
<img width="287" alt="broadmoor-driving-duration-scatter" src="https://user-images.githubusercontent.com/1664407/124358291-7a84b800-dbd4-11eb-998f-a9d475939863.png">
There's a ton of noise from drivers -- some short trips get pretty long, and some long trips get pretty short. Much (but not all) of this is due to parking. Because it's so scarce, some drivers spend lots of time looking for parking, until it dominates their trip time.

If we use `--infinite_parking` both for the baseline results and the edits, the results look more understandable:
![Screenshot from 2021-07-03 08-04-10](https://user-images.githubusercontent.com/1664407/124358465-4958b780-dbd5-11eb-8940-87ca5cee8a28.png)
There are still some driving trips that're dramatically affected:
![Screenshot from 2021-07-03 08-05-00](https://user-images.githubusercontent.com/1664407/124358494-68574980-dbd5-11eb-9d96-6628e05e9cd0.png)
But way less. I'll also dive into some of those to see why.

Parking scarcity and the tradeoffs of using street space for it are critical features of A/B Street, but they also require good data about the number of offstreet spots available per building and a good estimate on the total number of cars that need to park somewhere. We're making totally blind guesses about the first. The second is mostly a function of Soundcast, but since we don't distinguish car drivers and passengers, we might be trying to cram too many cars in. Any ideas for better off-street capacity estimates for Seattle specifically or in general?

And should we disable parking simulation for the purposes of the article? My experience in the area covered by the Arboretum map is that street parking is not hard to find, so it's pretty strange if our results are really sensitive to it.

**Comment by dabreegster** [comment]
There was a bug with infinite parking. Figuring out how to clean up the fix, but the results for drivers look way more expected -- the Broadmoor access edit with infinite parking basically doesn't affect trip times now:
![Screenshot from 2021-07-03 09-31-57](https://user-images.githubusercontent.com/1664407/124361031-b1ad9600-dbe1-11eb-89d6-20b7ffb210a6.png)

**Comment by michaelkirk** [comment]
My understanding is that due to vastly undercounting offstreet parking, we'll get more realistic results with `--infinite_parking`, and that I should re-run the results / prebake with that option. 

Is that right @dabreegster?

**Comment by dabreegster** [comment]
Correct. I started this work last week, then got bogged down trying to fix every last problem. (The "parking efficiency" layer is computed in a way that is often misleading -- the distance between a person and their car at any moment isn't useful to know; the distance from the next driving trip's start matters.)

I think we need to:
- decide how to mark a map or scenario as forcing infinite parking on
- prebake the results again
- sanity check what the outputs look like

I'm undecided where this bit flip should live. Either in the map or scenario will require rebuilding lots of files, so I don't want to change my mind in a few days. :)   I think in the short term, I might hardcode it based on map name. I'll start something right now and hopefully send it by tonight.

**Comment by dabreegster** [comment]
As expected, being able to skip parking saves tons of time. This compares (no map edits, infinite parking) to the baseline of (no map edits, normal parking):
![infparking_vs_prev_baseline](https://user-images.githubusercontent.com/1664407/125375694-5f394b80-e33e-11eb-9ceb-382827a96e79.png)

And the more relevant comparison, between (access broadmoor proposal, inf parking) and (no map edits, inf parking):
![Screenshot from 2021-07-12 18-25-58](https://user-images.githubusercontent.com/1664407/125375846-a1628d00-e33e-11eb-8182-39649fc3062b.png)
Deselecting driving only modifies some of the short trips:
![Screenshot from 2021-07-12 18-26-43](https://user-images.githubusercontent.com/1664407/125375934-cd7e0e00-e33e-11eb-83cd-07c8ea8a6465.png)
Just to be sure what's going on there, the effects just on drivers:
![Screenshot from 2021-07-12 18-27-40](https://user-images.githubusercontent.com/1664407/125375969-da026680-e33e-11eb-848d-080d2f2bece9.png)


And risk exposures:
![Screenshot from 2021-07-12 18-27-59](https://user-images.githubusercontent.com/1664407/125375998-e981af80-e33e-11eb-9915-2c9d59d393a6.png)
![Screenshot from 2021-07-12 18-28-05](https://user-images.githubusercontent.com/1664407/125376001-ea1a4600-e33e-11eb-8398-746838a16fb3.png)


I'll dig through some of the individual driving trips that got dramatically slower with the edits, but I think we should make this change and include it in the article. Then we don't have to justify the noise from bad data quality.

**Comment by michaelkirk** [comment]
> And risk exposures:

Are those risk exposures comparing:

A: "Finite parking w/o Broadmoor edits" vs "Inifinite parking w/ Broadmoor edits"
or
B: "Infinite parking w/o Broadmoor edits" vs "Inifinite parking w/ Broadmoor edits"

Either is fine, but interesting that a few arterial crossings went up.

**Comment by dabreegster** [comment]
B: "Infinite parking w/o Broadmoor edits" vs "Inifinite parking w/ Broadmoor edits"

Indeed weird. I guess we don't have the trip table equivalent for finding example trips, so not sure what's happening.
